### PR TITLE
cloudslang cli ui improvement

### LINKED ISSUE #928: cloudslang cli ui improvement

### PR DESCRIPTION
**Pull request by gituser173**
Fixes #928
Signed-off-by: Claudiu Ban claudiu-laurian.ban@hpe.com

### CODE REVIEW

**Comment by Bonczidai** [comment]
shouldn't we occasionally purge the `tasks` collection?

**Comment by lucian-cm** [comment]
@Bonczidai that is indeed required, at least on  each `waitForAllPrintTasksToFinish`()

File: cloudslang-cli/src/test/java/io/cloudslang/lang/cli/services/ConsolePrinterImplTest.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        Future<?> firstFuture = mock(Future.class);
+        Future<?> secondFuture = mock(Future.class);
+
+        tasks.add(firstFuture);
+        tasks.add(secondFuture);
+
+        consolePrinter.waitForAllPrintTasksToFinish();
+
+        assertEquals(0, tasks.size());
+        verify(firstFuture, times(1)).get(1, TimeUnit.MINUTES);
```
Comment (lucian-cm) [inline cloudslang-cli/src/test/java/io/cloudslang/lang/cli/services/ConsolePrinterImplTest.java]: times(1) is redunand in verify it verifies 1 time by default Please change

File: cloudslang-cli/src/main/java/io/cloudslang/lang/cli/services/SyncTriggerEventListener.java [hunk 2]
```diff
@@ -48,6 +49,9 @@
     private AtomicReference<String> errorMessage = new AtomicReference<>("");
     private boolean isDebugMode = false;
 
+    @Autowired
```
Comment (lucian-cm) [inline cloudslang-cli/src/main/java/io/cloudslang/lang/cli/services/SyncTriggerEventListener.java]: Cannot autowire here
See io.cloudslang.lang.cli.services.ScoreServicesImpl#triggerSync where we have new SyncTriggerEventListener()
We need to have a static method to print in the ConsolePrinter to access it from here with java method call.

**Comment by lucian-cm** [comment]
Looks good to me 👍

**Pull request merged**